client/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -38,10 +38,13 @@ const LoginForm: React.FC<Props> = props => {
   }
 
   const onLogin = () => {
-    const fieldErrors = LoginValidator.localValidate(email, password)
+    let fieldErrors = LoginValidator.localValidate(email, password)
+    if (invitedUser && invitedUser.status !== 'active') {
+      fieldErrors = LoginValidator.invitationValidate(email, password, password2)
+    }
```

Comment:
Can we use single row?
`const fieldErrors = LoginValidator.localValidate(email, password, password2)` and handle this in `LoginValidator.localValidate` ? you can initialize password2 to be undefined, and check if it's undefined before handling

Reply:
return { email: validateEmail(email), password: validatePassword(password), password2: password2 !== undefined ? validatePassword(password2) : null }